test/unit/SourceLocator/Ast/FindReflectionsInTreeTest.php

```diff
@@ -235,9 +235,8 @@ public function testInvokeCallsReflectNodesForNotGlobalDefine(): void
         $functionReflector->method('reflect')
             ->willReturn($mockFunctionReflection);
 
-        $strategy->expects($this->once())
-            ->method('__invoke')
-            ->willReturn($mockFunctionReflection);
+        $strategy->expects($this->never())
```

Comment:
This is quite a bold expectation change? :thinking:

Reply:
Yes, it's. The invoke is not called because now we don't create the `ReflectionFunction` instance from `function` node when we are looking for `constant` identifier.

And the `define` is invalid (not global `define`) so we don't call the `invoke` for the constant too.